x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -251,7 +250,15 @@ export class MBMapContainer extends React.Component {
     if (!this._isMounted) {
       return;
     }
-    ReactDOM.render((<FeatureTooltip properties={content} onCloseClick={this._onTooltipClose}/>), this._tooltipContainer);
+    const isLocked = this.props.tooltipState.type === TOOLTIP_TYPE.LOCKED;
+    ReactDOM.render((
+      <FeatureTooltip
+        properties={content}
+        closeTooltip={this._onTooltipClose}
+        shouldShowFilterButtons={this.props.isFilterable && isLocked}
```

Comment:
This can just be `showFilterButtons` just like `showCloseButton` is not "should ShowCloseButton"